azure-iot-device/azure/iot/device/provisioning/security/sk_security_client.py

```diff
@@ -0,0 +1,65 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""This module contains a client that is responsible for providing shared access tokens that will eventually establish
+ the authenticity of devices to Device Provisioning Service.
+"""
+
+from azure.iot.device.common.sastoken import SasToken
+
+
+class SymmetricKeySecurityClient(object):
+    """
+    A client that is responsible for providing shared access tokens that will eventually establish
+    the authenticity of devices to Device Provisioning Service.
+    """
+
+    def __init__(self, registration_id, symmetric_key, id_scope):
```

Comment:
>registration_id, symmetric_key, id_scope [](start = 23, length = 40)

Carter is going to tell you to make these into keyword args...

Reply:
according to the guidelines of azure SDKs it says `DO specify the parameter name when calling methods with more than two required positional parameters` . It seems this is only during calling and not during definition. 

---
In reply to: [276842803](https://github.com/Azure/azure-iot-sdk-python-preview/pull/81#discussion_r276842803) [](ancestors = 276842803)